### PR TITLE
Missing reference expansions part 2

### PR DESCRIPTION
**Pull request by schleichardt**
@lauraluiz please review

### CODE REVIEW

File: sphere-common/src/main/java/io/sphere/sdk/queries/IntermediateQuerySort.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package io.sphere.sdk.queries;
+
+import io.sphere.sdk.models.Base;
+
+import static io.sphere.sdk.queries.QuerySortDirection.*;
+
+/** A sort model to decide the direction. */
+public class IntermediateQuerySort<T> extends Base {
```
Comment (lauraluiz) [inline sphere-common/src/main/java/io/sphere/sdk/queries/IntermediateQuerySort.java]: Suggestion (not necessarily better): `DirectionlessQuerySort`

File: sphere-models/src/main/java/io/sphere/sdk/orders/expansion/OrderExpansionModel.java [hunk 2]
```diff
@@ -11,20 +10,21 @@
...
     }
 
-    public static OrderExpansionModel<Order> of() {
-        return new OrderExpansionModel<>();
+    public SyncInfoExpansionModel<T> syncInfo() {
+        //since it is a set, there is no method with index access
+        return new SyncInfoExpansionModel<>(pathExpressionOption(), "syncInfo[*]");
     }
 
-    public ExpansionPath<T> customerGroup() {
```
Comment (lauraluiz) [inline sphere-models/src/main/java/io/sphere/sdk/orders/expansion/OrderExpansionModel.java]: Is the customer group not expandable anymore?

Comment (schleichardt) [inline sphere-models/src/main/java/io/sphere/sdk/orders/expansion/OrderExpansionModel.java]: It is in the base class CartLikeExpansionModel, I wan't share the expansions of cart/order so I don't need to write too much tests.

Comment (lauraluiz) [inline sphere-models/src/main/java/io/sphere/sdk/orders/expansion/OrderExpansionModel.java]: Thanks, I missed that :P

**Comment by lauraluiz** [comment]
You can merge :+1:

**Pull request merged**